apps/cms/src/api/test/content-types/test/schema.json

```diff
@@ -0,0 +1,49 @@
+{
+  "kind": "collectionType",
+  "collectionName": "tests",
+  "info": {
+    "singularName": "test",
+    "pluralName": "tests",
+    "displayName": "Test",
+    "description": ""
+  },
+  "options": {
+    "draftAndPublish": true
+  },
+  "pluginOptions": {},
+  "attributes": {
+    "title": {
+      "type": "string",
+      "required": true,
+      "unique": true
+    },
+    "passingScore": {
+      "type": "decimal",
+      "required": true,
+      "min": 0,
+      "max": 100,
+      "default": 0
+    },
+    "status": {
+      "type": "enumeration",
+      "enum": [
+        "draft",
+        "published"
+      ],
+      "default": "draft",
+      "required": true
+    },
+    "timeLimit": {
+      "type": "time",
+      "required": true,
+      "default": "00:00"
+    },
+    "questions": {
+      "type": "dynamiczone",
+      "components": [
```

Comment:
Wait for this PR to be merged https://github.com/xponvn/test-assessment/pull/2. we will use question.choice-question and question.text-answer-question for components. like this
`
"components": [
        "question.choice-question",
        "question.text-answer-question"
 ]
`